### DISCUSSION TITLE
JBD SoC is displayed by the Cerbo GX, but 25% too low

### DISCUSSION THREAD

**Discussion by UnWitt** [mr-manuel/venus-os_dbus-serialbattery#134] [Q&A] [answered]
Hello,

the SoC of our two battery packs are passed on, but 25% too low.
So if the pack has 100%, 75% is displayed, at 95% it is 73.1%.

Of course, BatteryLife cannot work like this.
Can anyone help?

Dirk

Cerbo GX Firmware 3.52
JBD-AP21S002-L21S-200A-B-U-R-C (2 x 16S)

**Comment by UnWitt** [comment, marked as answer]
Hello,
Surprise: An update has solved the problem.
@ mr_manuel:
I have financed a beer for you.
Not just any beer, but a beer at the Wies'n, Munich (without traveling).

**Reply by mr-manuel** [reply]
Thanks
